config-connector/solutions/networking/helm/autoneg/README.md

```diff
@@ -0,0 +1,142 @@
+# Autoneg
+
+==================================================
+
+## NAME
+
+  autoneg
+
+## SYNOPSIS
+
+Autoneg solution uses network endpoint groups to provision load balancing across multiple clusters.
+This solution uses `./cluster/templates/autoneg.yaml` from [gke-autoneg-controller](https://github.com/GoogleCloudPlatform/gke-autoneg-controller).
+For demonstration purposes it uses a docker image with a simple Node.js service (bulankou/node-hello-world) with a single endpoint that prints out a message.
+
+## CONSUMPTION
+
+  1. Clone GoogleCloudPlatform/cloud-foundation-toolkit repository
+  
+      ```bash
+      git clone https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+      ```
+
+  1. Go to the autoneg folder:
+
+      ```bash
+      cd cloud-foundation-toolkit/config-connector/solutions/networking/helm/autoneg
+      ```
+
+## REQUIREMENTS
+
+1. GKE Cluster with Config Connector
+1. [Helm](../../../README.md#helm)
+
+## USAGE
+
+All steps are run from this directory.
+
+1. Review and update the values in `./lb/values.yaml`. Note that if you change cluster name and locaiton, you will need to change how they are used in `gcloud container clusters get-credentials` commands below. [PROJECT_ID] refers to the project where all the GCP resources will be created.  
+
+1. Configure clusters and load balancing resources with Helm
+
+    ```bash
+    # validate your chart
+    helm lint ./lb/ --set projectId=[PROJECT_ID]
+
+    # check the output of your chart
+    helm template ./lb/ --set projectId=[PROJECT_ID]
+
+    # install your chart
+    helm install ./lb/ --set projectId=[PROJECT_ID] --generate-name
```

Comment:
I got the following error when I run the solution. Is there anything wrong with the role name?

```
Error: IAMPolicyMember.iam.cnrm.cloud.google.com "iampolicymember-autoneg-custom" is invalid: []:
 Invalid value: map[string]interface {}{"apiVersion":"iam.cnrm.cloud.google.com/v1beta1", 
"kind":"IAMPolicyMember", "metadata":map[string]interface {}{"creationTimestamp":"2020-06-
09T18:35:26Z", "generation":1, "name":"iampolicymember-autoneg-custom", "namespace":"
[PROJECT_ID]", "uid":"fc9be478-aa7f-11ea-9c6f-42010a800076"}, "spec":map[string]interface {}
{"member":"serviceAccount:autoneg-system@[PROJECT_ID].iam.gserviceaccount.com", 
"resourceRef":map[string]interface {}{"apiVersion":"resourcemanager.cnrm.cloud.google.com/v1beta1", 
"external":"projects/[PROJECT_ID]", "kind":"Project"}, "role":"projects/[PROJECT_ID]/roles/autonegrole"}}: 
validation failure list:
spec.role in body should match '^roles/[\w\.]+$'
```

Reply:
Thanks. I added an item to requirements.